Tangem/Modules/ManageTokens/ManageTokensCoordinatorView.swift

```diff
@@ -13,18 +13,23 @@ struct ManageTokensCoordinatorView: CoordinatorView {
     @ObservedObject var coordinator: ManageTokensCoordinator
 
     var body: some View {
-        NavigationView {
-            if let model = coordinator.manageTokensViewModel {
-                ManageTokensView(viewModel: model)
-                    .navigationLinks(links)
+        ZStack {
+            NavigationView {
+                if let model = coordinator.manageTokensViewModel {
+                    ManageTokensView(viewModel: model)
+                }
             }
+            .navigationViewStyle(.stack)
+
+            sheets
```

Comment:
```suggestion
            NavigationView {
                if let model = coordinator.manageTokensViewModel {
                    ManageTokensView(viewModel: model)
                }
                
                sheets
            }
            .navigationViewStyle(.stack)
```
Мне кажется что так надо, хз, надо проверить
В `SwappingCoordinatorView` Андрей Ч сделал так

Reply:
> Мне кажется что так надо, хз, надо проверить В `SwappingCoordinatorView` Андрей Ч сделал так

Поправил